lib/mpi/responses/parser_base.rb

```diff
@@ -40,25 +40,12 @@ def sanitize_edipi(edipi)
         sanitized_result
       end
 
-      def sanitize_participant_id(participant_id)
-        return if participant_id.nil?
+      def sanitize_id(id)
+        return if id.nil?
 
         # Get rid of non-digit characters like 'UNK'/'ASKU'
-        sanitized_result = participant_id.match(/\d+/)&.to_s
-        if sanitized_result != participant_id
-          Rails.logger.info "Participant id sanitized, was: '#{participant_id}' now: '#{sanitized_result}'."
-        end
-        sanitized_result
-      end
-
-      def sanitize_birls_id(birls_id)
-        return if birls_id.nil?
-
-        # Get rid of non-digit characters like 'UNK'/'ASKU'
-        sanitized_result = birls_id.match(/\d+/)&.to_s
-        if sanitized_result != birls_id
-          Rails.logger.info "Birls id sanitized, was: '#{birls_id}' now: '#{sanitized_result}'."
-        end
+        sanitized_result = id.match(/\d+/)&.to_s
+        Rails.logger.info "Id sanitized, was: '#{id}' now: '#{id}'." if sanitized_result != id
```

Comment:
I know I missed this in the last review, but this shouldn't be a common rails log. Its not a good practice to send sensitive data to generic system logs. We should make this a debug level log and only show this log if we turn on debug mode for this service. I will go research now but Johnny (previous engineer on the team) showed me a while back one of the debug logs that could be turned on through a code change. We certainly would want to do this through a flipper, but we may want at least settle for not logging the real id to rails logger. This applies to all rails logger statements in this PR.

Reply:
[Here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/14921#issuecomment-715360767) is the example from Johnny